tests/network/sriov.go

```diff
@@ -343,12 +343,15 @@ var _ = Describe(SIG("SRIOV", Serial, decorators.SRIOV, func() {
 					initialGuestMemory      = "1Gi"
 					updatedGuestMemory      = "3Gi"
 					sriovNetworkLogicalName = "sriov-network"
+					mac                     = "86:4b:dc:31:e5:c8"
```

Comment:
please consider using dead beef as we are always doing, easier to detect when looking on logs and such